src/runtime/arch_tinygoriscv64.go

```diff
@@ -0,0 +1,92 @@
+// +build tinygo.riscv64
+
+package runtime
+
+import "device/riscv"
+
+const GOARCH = "arm64" // riscv pretends to be arm
```

Comment:
Could we use `GOARCH = "riscv64"` here?

Reply:
@jaddr2line or @aykevl ?

Reply:
Unfortunately not, if we want to retain support for older Go versions. And honestly the `GOARCH` value doesn't matter a whole lot in practice.